docs/platform/howto/add-storage-space.rst

```diff
@@ -24,12 +29,12 @@ You can add storage to your running service in `Aiven Console <https://console.a
 
 #. In the **Upgrade service storage** window, use the slider to add disk storage.
 
-   ..note::
+#. Select **Save changes**.
+
+.. note::
```

Comment:
This note is needed below step 4 I'm afraid. We want them to learn the info on the price before they save the changes. So most likely you need to number the steps explicitly (1-5), no using `#`.
The same way this is done for the "Remove added storage" section.

Reply:
Make sense! Thanks!